_pages/ato/ssp.md

```diff
@@ -0,0 +1,90 @@
+---
+title: System Security Plan
+---
+
+As described in [the NIST guide](http://csrc.nist.gov/publications/nistpubs/800-18-Rev1/sp800-18-Rev1-final.pdf#page=7):
+
+> The purpose of the system security plan is to provide an overview of the security requirements of the system and describe the controls in place or planned for meeting those requirements.
+
+GSA provides a [Word template](http://www.gsa.gov/graphics/staffoffices/System_Security_Plan_Template_072312_508.docx), but for 18F's purposes, we create a machine-readable version.
+
+We are working on a tool to help generate SSPs – check out [Compliance Masonry](https://github.com/opencontrol/compliance-masonry).
+
+### Template
+
+Add the following to your repository as `system-security-plan.yml`, with the information in the `<angle brackets>` filled in. Note that there are `TODO`s in some places where we need more documentation – for now, reach out to Noah Kunin for help with what should be filled in there.
+
+```yaml
+---
+name: <project name>
+# this should match the infrastructure Tag
+uniqueID: <partner-acronym>-<beginning-of-agreement>-<end-of-agreement>-<iaa-number>
+version: 0.1.0
+phase: <discovery|alpha|beta|live>
+information-types:
+- <TODO>
```

Comment:
That comes from NIST SP800-60 (http://csrc.nist.gov/publications/nistpubs/800-60-rev1/SP800-60_Vol1-Rev1.pdf). 

`D26 Civilian Operations` is what I see in most of our SSPs. D.26 is 'Direct Services for Civilians'.